cmd/web-handlers.go

```diff
@@ -570,11 +570,13 @@ func (web *webAPIHandlers) ListObjects(r *http.Request, args *ListObjectsArgs, r
 	nextMarker := ""
 	// Fetch all the objects
 	for {
-		// Limit browser to 1000 batches to be more responsive, scrolling friendly.
-		lo, err := listObjects(ctx, args.BucketName, args.Prefix, nextMarker, SlashSeparator, 1000)
+		// Limit browser to defaults batches to be more responsive, scrolling friendly.
+		lo, err := listObjects(ctx, args.BucketName, args.Prefix, nextMarker, SlashSeparator, -1)
 		if err != nil {
 			return &json2.Error{Message: err.Error()}
 		}
+
```

Comment:
on the browser, it stops listing at 1000 keys. Isn't it supposed to use next marker and iterate till the end?

Reply:
it doesn't stop @poornas you need to build from this PR